tests/tools/oecert/host/host.cpp

```diff
@@ -195,53 +193,47 @@ static oe_result_t _gen_report(
             }
         }
 
-        // Check if collaterals need to be generated
-        if (_params.gen_collaterals)
-        {
-            char collateral_filename[1024 + 1];
+        char collateral_filename[1024 + 1];
 
-            if (strlen(report_filename) < (1024 - 4))
+        if (strlen(report_filename) < (1024 - 4))
+        {
+            uint8_t* collaterals = NULL;
+            size_t collaterals_size = 0;
+            oe_report_header_t* header = (oe_report_header_t*)remote_report;
+
+            sprintf(collateral_filename, "%s.col", report_filename);
+            printf("Generatting collateral file: %s\n", collateral_filename);
+
+            result = oe_get_sgx_endorsements(
+                header->report,
+                header->report_size,
+                &collaterals,
+                &collaterals_size);
+            if (result != OE_OK)
             {
-                uint8_t* collaterals = NULL;
-                size_t collaterals_size = 0;
-                oe_report_header_t* header = (oe_report_header_t*)remote_report;
-
-                sprintf(collateral_filename, "%s.col", report_filename);
-                printf(
-                    "Generatting collateral file: %s\n", collateral_filename);
-
-                result = oe_get_collaterals_internal(
-                    header->report,
-                    header->report_size,
-                    &collaterals,
-                    &collaterals_size);
-                if (result != OE_OK)
-                {
-                    printf("ERROR: Failed to get collaterals\n");
-                    goto exit;
-                }
-
-                // TODO: Current collateral structure is not self contained.
-                // Needs to be updated to be serialisable.
-                //
-                FILE* col_fp = fopen(collateral_filename, "wb");
-                if (!col_fp)
-                {
-                    printf(
-                        "Failed to open collateral file %s\n",
-                        collateral_filename);
-                    result = OE_FAILURE;
-                    goto exit;
-                }
-                fwrite(collaterals, collaterals_size, 1, col_fp);
-                fclose(col_fp);
-                printf("collaterals_size = %zu\n", collaterals_size);
+                printf("ERROR: Failed to get collaterals\n");
+                goto exit;
             }
-            else
+
+            // TODO: Current collateral structure is not self contained.
```

Comment:
Is there an Issue # filed to track this TODO?  (If so, would be nice to reference the # in the code comment)

Reply:
Good idea.  Will keep that in mind.  This is still WIP.